packages/cli/src/commands/pull.ts

```diff
@@ -83,43 +77,6 @@ function parseArgs(client: Client) {
   return argv;
 }
 
-type LinkResult = {
-  org: Org;
-  project: Project;
-};
-async function ensureLink(
-  client: Client,
-  cwd: string,
-  yes: boolean
-): Promise<LinkResult | number> {
-  let link = await getLinkedProject(client, cwd);
-  if (link.status === 'not_linked') {
-    link = await setupAndLink(client, cwd, {
-      autoConfirm: yes,
-      successEmoji: 'link',
```

Comment:
Same here: it’s a different successEmoji

Reply:
`successEmoji` now defaults to `'link'`.